pkg/apis/imageregistry/v1alpha1/types.go

```diff
@@ -10,7 +10,10 @@ import (
 )
 
 const (
-	OperatorStatusTypeRemoved = "Removed"
+	OperatorStatusTypeRemoved             = "Removed"
+	OperatorNamespace                     = "openshift-image-registry"
```

Comment:
No. Use: 
https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/client/client.go#L65-L74

Reply:
Did you mean https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/client/client.go#L56 to get the namespace?

Reply:
Fixed in next push.